text/0617-rfc-stages.md

```diff
@@ -0,0 +1,409 @@
+of the relevant package(s). If there are any critical deviations from the original RFC, 
+they are briefly noted at the top of the RFC.
+
+If the work for an RFC is spread across multiple releases of Ember or other packages, 
+the RFC is considered to be in the Released stage when all features are available in
+stable releases and those packages and versions are noted in the RFC frontmatter.
+
+Ember's RFC process can be used for process and work plans that are not about code. 
+Some examples include Roadmap RFCs, this RFC itself, and changes to learning resources.
+When such an RFC is a candidate for Released, the work should be shipped as described, 
+and the result should presented to the team with the intent of gathering feedback 
+about whether anything is missing. If there is agreement that the work is complete, 
+the RFC may be marked "Released" and a date is provided instead of a version.
+
+An RFC is moved into "Released" when the above is verified by consensus of the 
+relevant team(s) via PR to update the stage.
+
+#### Recommended
+
+The "Recommended" stage is the final milestone for an RFC. It provides a signal 
+to the wider community to indicate that a feature has been put through its 
+ecosystem paces and is ready to use.
+
+The "Recommended" stage is most important for suites of features that are designed 
+as a number of separate RFCs. It allows the Ember maintainers to stabilize individual 
+features once they are technically feature complete, an important goal for maintaining 
+technical velocity.
+
+To reach the "Recommended" stage, the following should be true:
+
+- If appropriate, the feature is integrated into the tutorial and the guides prose.
+API documentation is polished and updates are carried through to other areas of 
+API docs that may not directly pertain to the feature.
+- If the proposal replaces an existing feature, the addon ecosystem has largely
+updated to work with both old and new features. 
+- If the proposal updates or replaces an existing feature, high-quality codemods are 
+available
+- If needed, Ember debugging tools as well as popular IDE support have been
+updated to support the feature.
+- If the feature is part of a suite of features that were designed to work together
+for best ergonomics, the other features are also ready to be "Recommended".
+- Any criteria for "Recommended" for this proposal that were established in the 
+[Ready For Release](#Ready-for-Release) stage have been met.
+
+An RFC is moved into "Recommended" via PR to update the stage. An FCP is required
+to enter this stage. Multiple RFCs may be moved as a batch into "Recommended" with
+the same PR.
+
+#### Closed
+
+A [Proposed](#Proposed) RFC may be closed after an FCP period. This is the same 
+as the existing process. A closed RFC is discontinued.
+
+#### Discontinued
+
+An previously [Accepted](#Accepted) RFC may be discontinued at any point. The RFC
```

Comment:
Neither _closed_ nor _discontinued_ seem to apply for an RFC, which is in _exploring_ stage. Did I missed something?

Reply:
Good catch -- it was intended that Exploring RFCs can move to closed via FCP, same as proposed. I'll update.